fluentinterface/src/main/java/com/iluwatar/fluentinterface/App.java

```diff
@@ -0,0 +1,110 @@
+package com.iluwatar.fluentinterface;
+
+import com.iluwatar.fluentinterface.fluentiterable.lazy.LazyFluentIterable;
+import com.iluwatar.fluentinterface.fluentiterable.simple.SimpleFluentIterable;
+
+import java.util.*;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+import static java.lang.String.valueOf;
+
```

Comment:
Add documentation guiding the reader on how to understand this class and pattern. Refer documentation in `App.java` of other patterns.